internal/editor/editor.go

```diff
@@ -128,3 +86,25 @@ func Invoke(ctx context.Context, editor string, content []byte) ([]byte, error)
 
 	return nContent, nil
 }
+
+func vimOptions(editor string) []string {
+	if editor != "vi" && editor != "vim" && editor != "neovim" {
+		return []string{}
+	}
+
+	path := "/dev/shm/gopass*"
+	if runtime.GOOS == "darwin" {
```

Comment:
How does that work on Windows? (Are we even using vim on windows?)

Reply:
I have no idea. Someone else would need to test that.